docker/build/x86_64/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # Build Frontend
 FROM node:alpine as frontend
-RUN apk add --no-cache make
+RUN apk add --no-cache make git
```

Comment:
Might have missed copying this change over to the CUDA Dockerfile as well (`docker/build/x86_64/Dockerfile-CUDA`)